functional_tests.go

```diff
@@ -11285,22 +11285,20 @@ func testRemoveObjects() {
 	var reader = getDataReader("datafile-129-MB")
 	defer reader.Close()
 
-	n, err := c.PutObject(context.Background(), bucketName, objectName, reader, int64(bufSize), minio.PutObjectOptions{})
+	_, err = c.PutObject(context.Background(), bucketName, objectName, reader, int64(bufSize), minio.PutObjectOptions{})
 	if err != nil {
 		log.Fatalln(err)
```

Comment:
Thanks!

Could you replace log.Fatalln(err) with logError(testName, function, args, startTime, "", "Error uploading object", err) here and in the next few lines ?